MapleServer2/Packets/ClubPacket.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Types;
+
+namespace MapleServer2.Packets
+{
+    public static class ClubPacket
+    {
+        public static Packet CreateClub(Player player, Player partyLeader, string clubName, List<Player> members)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.CLUB);
+            pWriter.WriteByte(0x2); // mode
```

Comment:
I would maybe use enums/constants for this if the modes continue but for now this is good.